.github/workflows/ci.yml

```diff
@@ -17,6 +17,7 @@ jobs:
           - {VERSION: "3.8", NOXSESSION: "tests"}
           - {VERSION: "3.13", NOXSESSION: "tests"}
           - {VERSION: "3.13t", NOXSESSION: "tests"}
+          - {VERSION: "pypy-3.11", NOXSESSION: "tests-nocoverage"}
```

Comment:
Would you mind moving this to the linux section below (next too pypy 3.9 and 3.10)

Reply:
also nox session should be `tests`, we don't have a nocoverage variant (which is fine)

Reply:
all right 🙏